### DISCUSSION TITLE
Geräte import von CCU3 (Devices with incorrect configuration)

### DISCUSSION THREAD

**Discussion by jm3west** [Zefau/ioBroker.jarvis#1903] [General]
Moin,

hab schon im ioBroker-Forum die Frage gestellt.
Wurde dann hierher verwiesen.

Habe gerade noch was gefunden ....

Beim importieren von der CCU3 sind bei mir 22 Geräte (Devices with incorrect configuration)
Es handelt sich überwiegend um HmIP-FSM16

Komischerweise lassen sich die vorgenannten Geräte, in der CCU3, steuern.

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/Zefau/ioBroker.jarvis/files/9640127/Bildschirmfoto.2022-09-25.um.07.57.37.pdf">
  <source media="(prefers-color-scheme: light)" srcset="https://github.com/Zefau/ioBroker.jarvis/files/9640127/Bildschirmfoto.2022-09-25.um.07.57.37.pdf">
  <img alt="Shows an illustrated sun in light color mode and a moon with stars in dark color mode." src="https://github.com/Zefau/ioBroker.jarvis/files/9640127/Bildschirmfoto.2022-09-25.um.07.57.37.pdf">
</picture>

**Comment by Zefau** [comment]
Jarvis kann nur Geräte mit bekannter Struktur importieren. Ich muss die quasi vorher anlernen.

Schick mir gerne von den nicht importierbaren Geräten den Objektbaum als JSON Export und sag mir, welche Datenpunkte davon durch den Importer erkannt werden sollen.

**Comment by jm3west** [comment]
… nachfolgend, im JSON-Format, die Geräte die nicht importiert wurden



> Am 25.09.2022 um 11:57 schrieb Zefau ***@***.***>:
> 
> 
> Jarvis kann nur Geräte mit bekannter Struktur importieren. Ich muss die quasi vorher anlernen.
> 
> Schick mir gerne von den nicht importierbaren Geräten den Objektbaum als JSON Export und sag mir, welche Datenpunkte davon durch den Importer erkannt werden sollen.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/Zefau/ioBroker.jarvis/discussions/1903#discussioncomment-3726823>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AGAA47EXR67XSXJHSBJJ25TWAAOYDANCNFSM6AAAAAAQU6HSTY>.
> You are receiving this because you authored the thread.
> 


HmIP-Gerät:	HmIP-FSM16

{
  "_id": "hm-rpc.1.000B9BE99XXXXX",
  "type": "device",
  "common": {
    "name": "HA Vorgarten - Wandlicht",
    "icon": "/icons/135_hmip-fsm16_thumb.png"
  },
  "native": {
    "TYPE": "HmIP-FSM16",
    "SUBTYPE": "FSM",
    "ADDRESS": "000B9BE99XXXXX",
    "RF_ADDRESS": 3009536,
    "CHILDREN": [
      "000B9BE99XXXXX:0",
      "000B9BE99XXXXX:1",
      "000B9BE99XXXXX:2",
      "000B9BE99XXXXX:3",
      "000B9BE99XXXXX:4",
      "000B9BE99XXXXX:5",
      "000B9BE99XXXXX:6",
      "000B9BE99XXXXX:7"
    ],
    "PARENT": "",
    "PARENT_TYPE": "",
    "INDEX": 0,
    "AES_ACTIVE": 1,
    "PARAMSETS": [
      "MASTER",
      "SERVICE"
    ],
    "FIRMWARE": "1.22.8",
    "AVAILABLE_FIRMWARE": "0.0.0",
    "UPDATABLE": true,
    "FIRMWARE_UPDATE_STATE": "UP_TO_DATE",
    "VERSION": 4,
    "FLAGS": 1,
    "LINK_SOURCE_ROLES": "",
    "LINK_TARGET_ROLES": "",
    "DIRECTION": 0,
    "GROUP": "",
    "TEAM": "",
    "TEAM_TAG": "",
    "TEAM_CHANNELS": [],
    "INTERFACE": "",
    "ROAMING": 0,
    "RX_MODE": 0
  },
  "from": "system.adapter.hm-rega.0",
  "user": "system.user.admin",
  "ts": 1664087538273,
  "acl": {
    "object": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  }
}

HmIP-Gerär:	HmIP-MOD-HO

{
  "_id": "hm-rpc.1.00241BE99XXXXX",
  "type": "device",
  "common": {
    "name": "GA Garagentor - Antrieb",
    "icon": "/icons/191_hmip-mod-ho_thumb.png"
  },
  "native": {
    "TYPE": "HmIP-MOD-HO",
    "SUBTYPE": "MOD-HO",
    "ADDRESS": "00241BE99XXXXX",
    "RF_ADDRESS": 2794476,
    "CHILDREN": [
      "00241BE99XXXXX:0",
      "00241BE99XXXXX:1",
      "00241BE99XXXXX:2"
    ],
    "PARENT": "",
    "PARENT_TYPE": "",
    "INDEX": 0,
    "AES_ACTIVE": 1,
    "PARAMSETS": [
      "MASTER",
      "SERVICE"
    ],
    "FIRMWARE": "1.0.14",
    "AVAILABLE_FIRMWARE": "0.0.0",
    "UPDATABLE": true,
    "FIRMWARE_UPDATE_STATE": "UP_TO_DATE",
    "VERSION": 1,
    "FLAGS": 1,
    "LINK_SOURCE_ROLES": "",
    "LINK_TARGET_ROLES": "",
    "DIRECTION": 0,
    "GROUP": "",
    "TEAM": "",
    "TEAM_TAG": "",
    "TEAM_CHANNELS": [],
    "INTERFACE": "",
    "ROAMING": 0,
    "RX_MODE": 0
  },
  "from": "system.adapter.hm-rega.0",
  "user": "system.user.admin",
  "ts": 1635684635741,
  "acl": {
    "object": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  }
}

HmIP-Heizgruppe

{
  "_id": "hm-rpc.2.INT0000001",
  "type": "device",
  "common": {
    "name": "OG Loft Heizung INT0000001",
    "icon": "/icons/175_hmip-group-heating_thumb.png"
  },
  "native": {
    "TYPE": "HmIP-HEATING",
    "SUBTYPE": "",
    "ADDRESS": "INT0000001",
    "RF_ADDRESS": 0,
    "CHILDREN": [
      "INT0000001:0",
      "INT0000001:1",
      "INT0000001:2",
      "INT0000001:3",
      "INT0000001:4",
      "INT0000001:5",
      "INT0000001:6"
    ],
    "PARENT": "",
    "PARENT_TYPE": "",
    "INDEX": 0,
    "AES_ACTIVE": 0,
    "PARAMSETS": [
      "MASTER"
    ],
    "FIRMWARE": "2.0.0",
    "AVAILABLE_FIRMWARE": "2.0.0",
    "UPDATABLE": true,
    "FIRMWARE_UPDATE_STATE": "",
    "VERSION": 131072,
    "FLAGS": 1,
    "LINK_SOURCE_ROLES": "",
    "LINK_TARGET_ROLES": "",
    "DIRECTION": 0,
    "GROUP": "",
    "TEAM": "",
    "TEAM_TAG": "",
    "TEAM_CHANNELS": [],
    "INTERFACE": "",
    "ROAMING": 0,
    "RX_MODE": 1
  },
  "from": "system.adapter.hm-rega.0",
  "user": "system.user.admin",
  "ts": 1663601703264,
  "acl": {
    "object": 1636,
    "owner": "system.user.admin",
    "ownerGroup": "system.group.administrator"
  }
}

**Reply by Zefau** [reply]
@jm3west sorry für die späte Rückmeldung. Sollte inzwischen via v3.1 importierbar sein.
